app/javascript/admin/table_with_details.js

```diff
@@ -36,22 +35,6 @@ $(document).ready(function() {
                     bSortable: false,
                     aTargets: [ 0 ]
                 },
-                {
-                    "targets": [ 1 ],
-                    "visible": false
-                },
-                {
-                    "targets": [ 2 ],
-                    "visible": false
-                },
-                {
-                    "targets": [ 3 ],
-                    "visible": false
-                },
-                {
-                    "targets": [ 4 ],
-                    "visible": false
-                }
```

Comment:
what are those and why they are no longer needed?

Reply:
Useless code I forgot to delete it last time.
It makes columns hidden, then used their data to fill the extended part of table. anyway, I used another approach.